HISTORY.md

```diff
@@ -44,3 +44,7 @@
  - Add `npmRelease` command
  - npm install the extension/views before release
  - Make release note extraction more robust
+
+## [Version 1.1.1]
```

Comment:
Should it be the same as in package.json 1.1.2 ?

Reply:
1.1.1 was already released with the bugfix. I'm considering that this PR, 1.1.2, does not need to be a released version and won't appear in the changelog.

Really I'm just adding/updating some non-code files, and bumping the version is just a formality.